src/transformers/models/auto/image_processing_auto.py

```diff
@@ -324,15 +324,15 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
         if image_processor_class is None and image_processor_auto_map is None:
             feature_extractor_class = config_dict.pop("feature_extractor_type", None)
             if feature_extractor_class is not None:
-                logger.warning(
+                logger.info(
```

Comment:
@sgugger so in the previous PR you said it might be better to leverage logger.warning_once here, but based on @amyeroberts' comment in this discussion: https://discuss.huggingface.co/t/error-finding-processors-image-class-loading-based-on-pattern-matching-with-feature-extractor/31890, would it make sense to use logger.info here instead?

Basically we won't show the warning anymore to the user, unless he has the logging info set to the INFO level.

Reply:
For the auto loading, `logger.warning_once` I think would be better :)

Reply:
Ok, but that ultimately will lead to the same behaviour as on `main` right now right? Given that people load an image processor only once.

Reply:
So we can leave it as a warning then.